config/routes.rb

```diff
@@ -2,5 +2,6 @@
 
   devise_for :users
 	root 'landing_page#index'
```

Comment:
How about adding a button to this `landing_pages#index` to create a new game? Then people will be able to start using this.